wordpress-mobile/WordPress-iOS
# Switch to saved posts when tapping "view all" on the saved post notice

Fixes #15514

To test:
- Go to the Reader
- Save a post for later
- tap on "view all" in the notice that appears at the bottom after saving, and make sure you are redirected to the "Saved" tab and the content matches your saved posts

PR submission checklist:
